tests/integration/DOM-test.js

```diff
@@ -184,3 +185,19 @@ test('Clicking modal overlay', function(assert) {
   });
 
 });
+
+test('Pressing escape', function(assert) {
+
+  visit('/');
+
+  showModal(templateName);
+
+  andThen(function() {
+    keyEvent(inspect('overlay'), 'keydown', escapeKeyCode);
```

Comment:
Yes, `keyEvent` is synchronous but i think `inspect` or `showModal` aren't because when calling in this way, the test rants about `inspect('overlay')` not being found.